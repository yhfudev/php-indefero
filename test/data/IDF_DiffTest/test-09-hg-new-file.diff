diff -r 000000000000 -r 5203a6d9dc87 a
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/a	Wed Mar 16 16:12:09 2011 +0100
@@ -0,0 +1,1 @@
+foo
