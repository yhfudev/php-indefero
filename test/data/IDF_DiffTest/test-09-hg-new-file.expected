<?php return array (
  'a' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '',
          1 => '1',
          2 => "foo\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '0',
          1 => '0',
        ),
        1 => 
        array (
          0 => '1',
          1 => '1',
        ),
      ),
    ),
  ),
);